sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedIterable.java

```diff
@@ -23,18 +23,18 @@
  *
  * {@codesnippet com.azure.core.http.rest.pagedIterable.iterableByPage.while}
  *
- * @param  <T> The type of value contained in this {@link IterableStream}.
+ * @param <T> The type of value contained in this {@link IterableStream}.
  * @see PagedResponse
  * @see IterableStream
  */
 public class PagedIterable<T> extends IterableStream<T> {
-    private final PagedFlux<T> pagedFlux;
+    private final PagedFluxBase<T, ? extends PagedResponse<T>> pagedFlux;
 
     /**
      * Creates instance given {@link PagedFlux}.
      * @param pagedFlux to use as iterable
      */
-    public PagedIterable(PagedFlux<T> pagedFlux) {
+    public <P extends PagedResponse<T>> PagedIterable(PagedFluxBase<T, P> pagedFlux) {
```

Comment:
Update javadocs to reflect the new input and output types.

Reply:
updated